aider/args.py

```diff
@@ -20,11 +20,17 @@ def get_parser(default_config_files, git_root):
         auto_env_var_prefix="AIDER_",
     )
     group = parser.add_argument_group("Main")
+    group.add_argument(
+        "--llm-history-file",
+        metavar="LLM_HISTORY_FILE",
+        default=".aider.llm.history",
```

Comment:
I think we want to default this to None? Most users won't want to be logging this level of detail.